package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint .",
     "flow": "flow check",
     "danger": "danger",
-    "fix": "eslint . --fix",
+    "fix": "yarn eslint . --fix",
```

Comment:
Ah, experimenting. Turns out all my failures where from files in the `build/` directory, so i added it to the ignore. Will undo.